### PR TITLE
Update bnb-chain.toml

### PR DESCRIPTION
**Pull request by STdevK**
Insert projects: Vooi, Cookie3, Cratos, InceptionLRT, NFTFeed, IGO dApp, Reclaim Protocol, Stability World AI,Rivo, iMe Messenger and Crypto Wallet, SERAPH In The Darkness, Grindery Wallet

### CODE REVIEW

File: data/ecosystems/b/bnb-chain.toml [hunk 1]
```diff
@@ -508,18 +519,27 @@ sub_ecosystems = [
...
+  "https://github.com/Igocoin",
   "https://github.com/liminal-market",
+  "https://github.com/locus-finance",
   "https://github.com/LuckyMachines",
   "https://github.com/Marketplace-of-the-Multiverse",
   "https://github.com/NFTin-App",
   "https://github.com/pandora-nft",
+  "https://github.com/reclaimprotocol",
+  "https://github.com/stabilityworld",
+  "https://github.com/vooi-app",
```
Comment (jubos) [inline data/ecosystems/b/bnb-chain.toml:542]: Since you added the sub ecosystems above, you don't also need to add the github orgs here.

**Review by jubos — CHANGES_REQUESTED** [review]

File: data/ecosystems/b/bnb-chain.toml (hunk 1 above)
Comment (STdevK) [reply data/ecosystems/b/bnb-chain.toml]: @jubos please check the changes

File: data/ecosystems/b/bnb-chain.toml [hunk 2]
```diff
@@ -506,22 +514,61 @@ sub_ecosystems = [
...
   "https://github.com/NFTin-App",
+  "https://github.com/Marketplace-of-the-Multiverse",
+  "https://github.com/Mercury-crew",
+  "https://github.com/micro3dev",
+  "https://github.com/MintStakeShare",
+  "https://github.com/miracleplay"
+  "https://github.com/moli0618",
+  "https://github.com/oooo-protocol",
   "https://github.com/pandora-nft",
+  "https://github.com/RayFinance",
```
Comment (jdesanto) [inline data/ecosystems/b/bnb-chain.toml:560]: This can be removed. It is an organization linked to BNB.

File: data/ecosystems/b/bnb-chain.toml [hunk 3]
```diff
@@ -506,22 +514,61 @@ sub_ecosystems = [
...
+  "https://github.com/Raw-Raw-Network",
+  "https://github.com/SleeplessAI-Lab",
+  "https://github.com/thelandlord-2023/thelandlord",
+  "https://github.com/Trendzme01",
+  "https://github.com/trump407",
+  "https://github.com/stabilityworld",
   "https://github.com/WebXCool",
-]
+  "https://github.com/WHY",
+  "https://github.com/VIALabs-io"
```
Comment (jdesanto) [inline data/ecosystems/b/bnb-chain.toml:569]: VIALabs-io already exists an organization with this repository. It is linked to BNB Chain.

**Review by jdesanto — CHANGES_REQUESTED** [review]
Left comments inline. Several repositories are empty. Several look like existing orgs linked to BNB.

File: data/ecosystems/b/bnb-chain.toml [hunk 4]
```diff
@@ -255,6 +258,7 @@ sub_ecosystems = [
   "Liminal Market",
   "Linear",
   "liqee",
+  "Locus.finance",
```
[the remark this answers is not in the document]
Comment (STdevK) [reply data/ecosystems/b/bnb-chain.toml:261]: Their app on dappbay is Rivo
https://dappbay.bnbchain.org/detail/rivo

File: data/ecosystems/b/bnb-chain.toml [hunk 5]
```diff
@@ -311,6 +316,7 @@ sub_ecosystems = [
   "OctoGamex",
   "Oddz",
   "Odin Protocol",
+  "Off Grid Platform",
```
[the remark this answers is not in the document]
Comment (STdevK) [reply data/ecosystems/b/bnb-chain.toml:319]: removed

File: data/ecosystems/b/bnb-chain.toml [hunk 6]
```diff
@@ -506,22 +514,61 @@ sub_ecosystems = [
...
   "https://github.com/Fexon-Finance",
+  "https://github.com/flap-sh",
+  "https://github.com/imemessenger",
+  "https://github.com/Karacuka",
   "https://github.com/label-platform",
+  "https://github.com/Igocoin",
+  "https://github.com/invluencer",
   "https://github.com/liminal-market",
+  "https://github.com/locus-finance",
+  "https://github.com/Lorenzo-Protocol",
```
[the remark this answers is not in the document]
Comment (STdevK) [reply data/ecosystems/b/bnb-chain.toml:549]: removed

File: data/ecosystems/b/bnb-chain.toml [hunk 7]
```diff
@@ -506,22 +514,61 @@ sub_ecosystems = [
...
 
 github_organizations = [
+  "https://github.com/AdriaGames",
+  "https://github.com/AIQ886",
+  "https://github.com/ArcX-world",
+  "https://github.com/Astra-Nova",
+  "https://github.com/Aura-Exchange",
+  "https://github.com/AutismAid",
+  "https://github.com/aveniumchain",
+  "https://github.com/BabyBakedBeans",
```
[the remark this answers is not in the document]
Comment (STdevK) [reply data/ecosystems/b/bnb-chain.toml:524]: removed

File: data/ecosystems/b/bnb-chain.toml [hunk 8]
```diff
@@ -506,22 +514,61 @@ sub_ecosystems = [
...
+  "https://github.com/BAIToken",
+  "https://github.com/Bivreost",
   "https://github.com/bnb-chain",
   "https://github.com/daohuntapp",
+  "https://github.com/chappyz",
+  "https://github.com/CookieCommunity",
+  "https://github.com/contextpilot",
```
[the remark this answers is not in the document]
Comment (STdevK) [reply data/ecosystems/b/bnb-chain.toml:531]: Yes, they are. You can find their dappbay profile below:
https://dappbay.bnbchain.org/detail/chappyz
https://dappbay.bnbchain.org/detail/cookie3
https://dappbay.bnbchain.org/detail/context-pilot

File: data/ecosystems/b/bnb-chain.toml (hunk 2 above)
Comment (STdevK) [reply data/ecosystems/b/bnb-chain.toml:560]: removed

File: data/ecosystems/b/bnb-chain.toml (hunk 3 above)
Comment (STdevK) [reply data/ecosystems/b/bnb-chain.toml:569]: removed VIALabs-io

**Comment by STdevK** [comment]
@jdesanto we attached their respective dappbay profile for reference, emtpy repositories are removed

**Review by mariashen — APPROVED** [review]
Thank you for the PR! To expedite the process, I suggest we approve this commit, but will make the following changes.

The following sub-ecosystems will be removed:
- Aave — supports 10+ chains, so categorized as EVM
- Grindery — remove, this should go under opBNB and I have put it there internally, it will be reflected on the next taxonomy export
- inceptionLRT — we don’t have this toml file and I do not see an open source repos for inception LRT
- NFTFeed — supports 10+ chains, so categorized as EVM
- Rehold — supports 10+ chains, so categorized as EVM

The following orgs will be removed:

- https://github.com/Bivreost — this seems like a general wallet and not BNB related

Accepted, but should be a sub-ecosystem. I’ve linked these separately and will be reflected in the next taxonomy export:
- https://github.com/MintStakeShare— I’ve created this as a sub ecosystem under BNB
- https://github.com/Astra-Nova — I’ve created this as a sub ecosystem under BNB
- https://github.com/miracleplay — I’ve created this as a sub ecosystem under BNB
- https://github.com/oooo-protocol — I’ve created this as a sub ecosystem under BNB

Remove -- these are users, not valid orgs (please submit the relevant repos in a different PR):
- https://github.com/AdriaGames
- https://github.com/AIQ886
- https://github.com/ArcX-world
- https://github.com/CratosTokenOfficial
- https://github.com/dev-four-d4
- https://github.com/imemessenger
- https://github.com/Igocoin
- https://github.com/Mercury-crew
- https://github.com/micro3dev
- https://github.com/moli0618
- https://github.com/SleeplessAI-Lab
- https://github.com/Trendzme01
- https://github.com/trump407
- https://github.com/stabilityworld

**Comment by jubos** [comment]
I made the adjustments suggested by @mariashen and manually fixed the conflicts / sort.

**Pull request closed without merging**